data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/external/ExternalEventResponseError.avsc

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "record",
+  "name": "ExternalEventResponseError",
+  "namespace": "net.corda.data.flow.event.external",
+  "doc": "Included in {@link ExternalEventResponse}s that have failed to be processed. The failure type may be further categorised by the {@link Error} enumeration.",
+  "fields": [
+    {
+      "name": "errorType",
+      "doc": "Error type",
+      "type": {
+        "name": "ExternalEventResponseErrorType",
+        "type": "enum",
+        "doc": "RETRY: A retriable error.\nPLATFORM: An error that should be propagated to the calling flow.\nFATAL: A fatal error.",
+        "symbols": [
+          "RETRY",
```

Comment:
Maybe TRANSIENT rather than RETRY to describe the type of error rather than the action to take?